### PR TITLE
Create math-katex filter

### PR DESCRIPTION
**Pull request by benabel**
Render math with katex server-side.

### CODE REVIEW

**Comment by tarleb** [comment]
Thank you for this! Looks like a very useful filter, and the code is beautifully clean, too.

We're currently discussing the future path of this repo, and I'd like to wait for the discussion to reach a conclusion before merging. Please feel free to add your voice there, too: #207

File: math-katex/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+DIFF ?= diff --strip-trailing-cr -u
+PANDOC ?= pandoc
+
+test: sample.md math-katex.lua expected.html
+	@$(PANDOC) \
```
Comment (tarleb) [inline math-katex/Makefile:5]: Makefiles are weird in that they *require* the first level indentation to be with tabs, not space.

Comment (benabel) [reply math-katex/Makefile:5]: It is already with tabs on my editor.

Comment (tarleb) [reply math-katex/Makefile:5]: Ah, OK, great. I probably viewed a previous version, or my browser/github didn't show the file correctly.

Comment (tarleb) [reply math-katex/Makefile:5]: Indeed, looks like GitHub renders the tabs with a width of a single space. I didn't know that.

**Comment by timtroendle** [comment]
@benabel, this looks very useful to me. Is your filter available somewhere?

**Comment by benabel** [comment]
@timtroendle in fact I never realized it you can find it on my fork https://github.com/benabel/lua-filters/tree/master/math-katex.

**Pull request closed without merging**